examples/03_connectivity/plot_confounds_removal.py

```diff
@@ -0,0 +1,170 @@
+"""
+Carefully defining the confounds
```

Comment:
Do you think "Confounds removal for functional connectivity analysis" sounds good ?